app/screens/settings/notification_settings_auto_responder/notification_settings_auto_responder.js

```diff
@@ -65,6 +65,22 @@ export default class NotificationSettingsAutoResponder extends PureComponent {
         this.saveUserNotifyProps();
     }
 
+    componentDidMount() {
+        setTimeout(() => {
+            requestAnimationFrame(() => {
```

Comment:
why a setTimeout and a requestAnimation frame?

Reply:
The requestAnimation actually allowed for a smoother action. I added a demo of how it responds when removed. It was about 50/50 on the keyboard flashing without the requestAnimation.

Reply:
But then why the setTimeout?

Reply:
I tested it with the following:

setTimeout only: Renders fine usually, but eery so often a stutter would appear. This could be resolved by extending the delay.

requestAnimation only: Rendered the keyboard immediately while navigation transition still occurred and still showed the flash.

Both together: Renders smoothly

I noticed we used a setTimeout with requestAnimation in another spot in hte code which is what caused me to try this method and it worked.